### PR TITLE
(feat) Add Khmer translations

### PR DESCRIPTION
**Pull request by kazlaw**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

### CODE REVIEW

**Comment by kazlaw** [comment]
Hi @vasharma05, @denniskigen  please have a look.

File: packages/esm-patient-labs-app/translations/km.json [hunk 1]
```diff
@@ -13,14 +13,14 @@
   "removeFromBasket": "Remove from basket",
   "resetTreeText": "Reset tree",
```
Comment (vasharma05) [inline packages/esm-patient-labs-app/translations/km.json:14]: Missing translation

File: packages/esm-patient-labs-app/translations/km.json [hunk 2]
```diff
@@ -13,14 +13,14 @@
...
   "resulted": "លទ្ធផល",
-  "resultsText": "results",
+  "resultsText": "លទ្ធផល",
   "returnToTimeline": "ត្រឡប់ទៅបន្ទាត់ពេលវេលា",
   "seeAllResults": "មើលលទ្ធផលទាំងអស់",
   "showTree": "Show tree",
```
Comment (vasharma05) [inline packages/esm-patient-labs-app/translations/km.json:19]: Mising translation

**Review by vasharma05 — APPROVED** [review]
Thanks @kazlaw !

**Pull request merged**